kafka/channel/README.md

```diff
@@ -125,14 +87,46 @@ kubectl get deployment -n knative-eventing kafka-webhook
 The Kafka Config Map is used to configure the `bootstrapServers` of your Apache
 Kafka installation:
 
-- **cluster-scoped**:
-
 ```shell
 kubectl get configmap -n knative-eventing config-kafka
 ```
 
-- **namespace-scoped**:
+### Namespace Dispatchers
 
-```shell
-kubectl get configmap   config-kafka
+By default events are received and dispatched by a single cluster-scoped dispatcher components. You can also specify whether events should be received and dispatched by the dispatcher in the same namespace as the channel definition by adding the
```

Comment:
can I mix ? 

have one cluster wide?

and a few ns specific channels ? 

(sure, all backed by the same kafka broker)

Reply:
yes you can mix and have a different Kafka Broker per NS